src/main/java/net/sacredlabyrinth/phaed/simpleclans/utils/ChatUtils.java

```diff
@@ -34,7 +34,17 @@
     private ChatUtils() {
     }
 
-    public static String parseColors(@NotNull String text) {
+    public static String parseColors(@Nullable String text) {
+        if (text == null) {
+            return "";
+        }
+
+        // Return chat color from hex
+        if (text.length() == 1) {
```

Comment:
Could this cause issues in chat messages?
Some people like to reply "k" meaning "ok".

Reply:
It's based on usages of parseColors.
![изображение](https://user-images.githubusercontent.com/42711312/124401960-83d96600-dd35-11eb-92ee-eb74b9b020e4.png)
So, probably, it would be the problem.

Reply:
Also if the clan tag's length is 1 (weird, but who knows), the tag would be parsed wrong.